swagger_py_codegen/templates/flask/validators.tpl

```diff
@@ -98,8 +98,8 @@ def response_filter(view):
     def wrapper(*args, **kwargs):
         resp = view(*args, **kwargs)
 
-        if isinstance(resp, current_app.response_class):
-            return resp
+        if isinstance(resp[0], current_app.response_class):
```

Comment:
`resp` can be a tuple, but also can be a response object. Reference index 0 without type checking maybe raise  `TypeError`.